src/database.rs

```diff
@@ -92,7 +92,7 @@ impl Database {
 
     pub fn insert_issues(&self, resp: Vec<GitHubIssue>, owner: &str, name: &str) -> Result<()> {
         for item in resp {
-            let keystr: String = format!("{}/{}#{}", owner, name, item.number);
+            let keystr: String = format!("{}/{}#{}by{}", owner, name, item.number, item.author);
```

Comment:
The tuple (owner, name, number) already identifies a single issue, and adding author to the key would only make it longer. It would be better to keep the author information in the value part.